src/utils/__tests__/articleHelper-test.ts

```diff
@@ -40,17 +47,6 @@ test('Random Urls should be false in isNDLAEmbedUrl function', async () => {
   expect(
     isNDLAEmbedUrl('this is a random string containing https://ndla.no'),
   ).toBe(false);
-
-  expect(
-    isNDLAEmbedUrl(
-      'https://staging.api.ndla.no/nb/subjects/subject:bfe6d0b2-fa5d-4962-84d7-796dedb099ff/topic:1:185608/resource:1:123925',
```

Comment:
Vet vi hvorfor denne url'en skulle være false før? Ser ut som en fornuftig url for meg?

Reply:
Kanskje fordi dette tidligere var en api-adresse.